components/tag-list/test/tag-list.visual-diff.js

```diff
@@ -32,6 +32,12 @@ describe('d2l-tag-list', () => {
 	describe('tag list item style behaviour', () => {
 
 		it('is correct on focus on tag list item', async function() {
+			await page.$eval('#default', async(elem) => {
```

Comment:
This test shows the keyboard tooltip, so if we're retrying we need to make sure that would still get shown